docs/guides/using_ocio/using_ocio.rst

```diff
@@ -0,0 +1,106 @@
+
+   Overrides the optimization settings being used by an application, for 
+   troubleshooting purposes.  The complete list of flags is in OpenColorTypes.h.
+
+
+.. include:: tool_overview.rst
+
+
+.. _faq-supportedlut:
+
+Supported LUT Formats
+=====================
+
+========== ===================================  =========================================
+Ext        Details                              Notes
+========== ===================================  =========================================
+3dl        Discreet legacy 3D-LUT               Read + Write Support.
+                                                Supports shaper + 3D-LUT
+ccc        ASC CDL ColorCorrectionCollection    Full read support.
+cc         ASC CDL ColorCorrection              Full read support.
+cdl        ASC CDL ColorCorrection              Full read support.
+clf        Academy/ASC Common LUT Format        Full read + write support.
+ctf        Autodesk Color Transform Format      Full read + write support.
+                                                Provides lossless serialization
+                                                of arbitrary OCIO transforms.
+csp        Cinespace (Rising Sun Research)      Read + Write Support.  Shaper is
+           LUT. Spline-based shaper LUT, with   resampled into simple 1D-LUT
+           either 1D or 3D-LUT.                 with 2^16 samples.
+cub        Truelight format. Shaper Lut + 3D    Full read support.
+cube       Iridas format. Either 1D or 3D Lut.  Read support.
+cube       Resolve format.                      Read support.
+lut        Discreet legacy 1D-LUT               Read support.
+hdl        Houdini. 1D-LUT, 3D-LUT, 1D shaper   Only 'C' type is supported.
+           Lut                                  Need to add R G B A RGB RGBA ALL.
+                                                No support for Sampling tag.
+                                                Header fields must be in strict order.
+icc/icm/pf ICC profile format                   Read support for basic monitor profiles.
+look       IRIDAS .look                         Read baked 3D-LUT embedded in file.
+                                                No mask support.
+mga/m3d    Pandora 3D-LUT                       Full read support.
+spi1d      1D-LUT format. Imageworks native     Full read support.
+           format.  HDR friendly, supports
+           arbitrary input and output domains
+spi3d      3D-LUT format. Imageworks native     Full read support.
+           format.
+spimtx     3x3 matrix + color offset.           Full read support.
+           Imageworks native matrix format.
+vf         Inventor 3D-LUT.                     Read support for 3D-LUT data
+                                                and global_transform element
+========== ===================================  =========================================
+
+.. note::
+   Shaper LUT application in OCIO currently only supports linear interpolation.
```

Comment:
Might be worth clarifying that this is referring to shaper sections of LUT files, and not to standalone shaper LUTs used in transforms.

Reply:
Good point.  I rewrote this paragraph.